programmers/문자열-내-p와-y의-게수.js

```diff
@@ -0,0 +1,18 @@
+function solution(s){
+    const word = s.split("");
+    const array = word.map(word => word.toLowerCase());
+    let p = 0;
+    let y = 0;
+    
+    for (let el of array) {
```

Comment:
```suggestion
    for (let el of s.toLowerCase()) {
```
for...of 에서 Array의 원소 뿐만 아니라 String의 문자를 개별 속성 값으로 이용할 수 있으므로 split("")을 이용하지 않으셔도 됩니다! 문자열 자체로도 사용하실 수 있어요. 또한 메소드 실행 할 때마다 복잡한 식이 아니라면 변수에 담지 않고 사용하셔도 괜찮습니다.